stdlib/Pkg/docs/src/index.md

```diff
@@ -455,24 +455,57 @@ shell> cat ~/.julia/packages/MbedTLS/h1Vu/deps/build.log
 
 So far we have added packages to the default project at `~/.julia/environments/v0.7`, it is, however, easy to create other, independent, projects.
 It should be pointed out if two projects uses the same package at the same version, the content of this package is not duplicated.
-This is done using the `init` command. Below we make a new directory and create a new project in it:
+In order to create a new project, create a directory for it and then activate that directory to make it the "active project" which package operations manipulate:
 
 ```
 shell> mkdir MyProject
 
 shell> cd MyProject
 /Users/kristoffer/MyProject
 
-(v0.7) pkg> init
-Initialized project at /Users/kristoffer/MyProject/Project.toml
+(v0.7) pkg> activate .
 
 (MyProject) pkg> st
     Status `Project.toml`
 ```
 
-Note that the REPL prompt changed when the new project was initiated, in other words, Pkg automatically set the current environment to the
-one that just got initiated. Since this is a newly created project, the status command show it contains no packages.
-Packages added here again in a completely separate environment from the one earlier used.
+Note that the REPL prompt changed when the new project is activated. Since this is a newly created project, the status command show it contains no packages, and in fact, it has no project or manifest file until we add a package to it:
+
+```
+shell> ls -l
+total 0
+
+(MyProject) pkg> add Example
+  Updating registry at `~/.julia/registries/Uncurated`
+  Updating git-repo `https://github.com/JuliaRegistries/Uncurated.git`
+ Resolving package versions...
+  Updating `Project.toml`
+  [7876af07] + Example v0.5.1
+  Updating `Manifest.toml`
+  [7876af07] + Example v0.5.1
+  [8dfed614] + Test
+
+shell> ls -l
+total 8
+-rw-r--r-- 1 stefan staff 207 Jul  3 16:35 Manifest.toml
+-rw-r--r-- 1 stefan staff  56 Jul  3 16:35 Project.toml
+
+hell> cat Project.toml
```

Comment:
This prompt is missing a character.

Reply:
> Imo PRs should be done against Pkg.jl as much as possible to avoid wasting CI time and to consolidate discussion.

Sequence of events:
1. originally wrote this against the main julia repo
2. realized it only touched Pkg code
3. tried to cherry-pick it over to the Pkg repo
4. failed after screwing around with it for ~30 minutes
5. gave up and made the RFC pull request here